content/en/docs/installation/authentication.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Authentication"
+weight: 40
+description: >
+  Guide to set up basic authentication with Fission
+---
+
+## Authentication in Fission
```

Comment:
Authentication for fission function invocation.

We should not highlight this as authentication for fission. All fission apis are still without auth.

We only do authentication for fission function invocation which may happen when router is exposed to internet.

Reply:
Updated the wording.